code/modules/mob/living/brain/life.dm

```diff
@@ -4,6 +4,17 @@
 		return
 	if(!loc)
 		return
+	// this is a sanity check to make sure that our MMI reference is still valid
+	// there used to be a bug where container still held a reference to an MMI that the brain wasn't
+	// inside of which caused the brain mob to be permanently stuck in nullspace even with a client,
+	// so this code exists as a failsafe in case everything goes wrong
+	if(!isnull(container))
+		if(!istype(container))
```

Comment:
`istype()` will implicitly check for `isnull()`, since null is typeless and will thus always fail `istype()`. I don't know if you can tweak the code with this new information but I figured it was worth telling you

Reply:
Oh ye, I already knew that, and this nullcheck is important because else we'd always runtime outside of a brain